### DISCUSSION TITLE
Renovate onboarding did not find all package.json files

### DISCUSSION THREAD

**Discussion by Demivan** [renovatebot/renovate#10228] [Request Help] [answered]
This is onboarding PR https://github.com/Demivan/fluent-vue/pull/601.

`package.json` files in `examples` folder are not being detected. I tried fixing it by adding: 
```json
{
  "matchPaths": ["+(examples/**/package.json)"],
  "rangeStrategy": "bump"
}
```
How do you specify where Renovate should look for package.json files?

**Comment by rarkins** [comment, marked as answer]
Adjust your `renovate.json` like this to include `ignorePresets`:

```json
{
  "extends": [
    "config:base",
    "group:allNonMajor"
  ],
  "ignorePresets": [":ignoreModulesAndTests"]
  ....
}
```

**Reply by Demivan** [reply]
Thank you. That worked.
